pkg/feature/domain/feature.go

```diff
@@ -77,6 +77,9 @@ func NewFeature(
 		VariationType: variationType,
 		CreatedAt:     time.Now().Unix(),
 		Maintainer:    maintainer,
+		Prerequisites: []*feature.Prerequisite{},
```

Comment:
The reason for adding an empty array instead of a null value is that the query becomes more complex because the `JSON_LENGTH` command for a `'null'` or a `["ABC"]` value is 1. Also, `'null'` is not the same as `NULL`.